ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14-presubmits.yaml

```diff
@@ -346,3 +346,61 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )openshift-pipelines-ocp4.16-lp-interop-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-v1\.14$
+    - ^release-v1\.14-
```

Comment:
OpenShift Pipelines 1.15 is latest stable release

Reply:
Will be updated on https://github.com/openshift/release/pull/54691
Thanks